src/toil/batchSystems/abstractBatchSystem.py

```diff
@@ -14,6 +14,7 @@
 
 
 from __future__ import absolute_import
+from collections import namedtuple
```

Comment:
Sorry, this is an artifact of having attempted a namedtuple and failed before moving to a  dict.